src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
+.wideImageOverride {
+    /* Center images. */
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+@media (orientation: portrait) {
+    .wideImageOverride {
+        width:100% !important;
+        max-width:100% !important;
+        height:auto !important;
+    }
+
+    /* Tablet override */
+    @media (min-device-width: 768px) {
```

Comment:
I think I'd be down with that - though probably a bit less so _no pun intended_ if that were the only LESS feature we used.